src/versions.js

```diff
@@ -36,7 +63,17 @@ function getVersion( resource, version ) { // jshint ignore:line
 	if ( version === undefined ) {
 		return resource;
 	} else {
-		return getVersions( resource )[ version ] || resource;
+		var versions = getVersions( resource );
+		version = parseInt( version );
```

Comment:
That's ok. You also can't prefix versions with "psketti" or a hex bitmap of Donald Trump. Unless there's some RFC or examples out there of people prefixing version numbers with 0s, I'mma just leave this be :grin: